### PR TITLE
fixes for default types as String & improved logging

### PR DESCRIPTION
**Pull request by ianturton**

### CODE REVIEW

File: modules/unsupported/wps/src/main/java/org/geotools/data/wps/WPSProcess.java [hunk 1]
```diff
@@ -124,20 +126,28 @@ protected WPSProcess(ProcessFactory factory) {
...
 			return null;
 		} catch (IOException e) {
+			LOGGER.info(e.getMessage());
 			return null;
 		}
 		
 		// if there is an exception in the response, return null
 		// TODO:  properly handle the exception?
-		if (response.getExceptionResponse() != null || response.getExecuteResponse() == null) {
+		if (response.getExceptionResponse() != null) {
```
Comment (aaime) [inline modules/unsupported/wps/src/main/java/org/geotools/data/wps/WPSProcess.java]: Wouldn't it be better to throw an exception instead?

**Comment by aaime** [comment]
Copyright headers not updated, no ticket reference, no test.

https://github.com/geotools/geotools/blob/master/CONTRIBUTING.md

**Comment by aaime** [comment]
Cough cough :-)

File: modules/unsupported/wps/src/main/java/org/geotools/data/wps/WPSProcess.java (hunk 1 above)
Comment (jodygarnett) [inline modules/unsupported/wps/src/main/java/org/geotools/data/wps/WPSProcess.java]: Throw exception, or at least change the logger level to "fine"

File: modules/unsupported/wps/src/main/java/org/geotools/data/wps/WPSProcess.java [hunk 2]
```diff
@@ -53,7 +55,7 @@
  * @source $URL$
  */
 public class WPSProcess extends AbstractProcess {
-
+	private static final Logger LOGGER = Logging.getLogger(WPSProcess.class.getName());
```
Comment (jodygarnett) [inline modules/unsupported/wps/src/main/java/org/geotools/data/wps/WPSProcess.java]: why use of getName() here?

**Comment by jodygarnett** [comment]
This change is too late for the 15-beta I am happy to see you working on this module.

**Comment by jodygarnett** [comment]
@ianturton ping?

**Pull request closed without merging**